fee_lab_to_nwb/scherrer_ophys/convert_session.py

```diff
@@ -1,28 +1,66 @@
 """Primary script to run to convert an entire session of data using the NWBConverter."""
 from pathlib import Path
+from datetime import datetime
+from zoneinfo import ZoneInfo
 
 from nwb_conversion_tools.utils import load_dict_from_file, dict_deep_update
-from fee_lab_to_nwb.scherrer_ophys import ScherrerOphysDatasetNWBConverter
 
-example_path = Path("D:/ExampleNWBConversion")
-example_session_id = example_path.stem
-nwbfile_path = example_path / f"{example_session_id}.nwb"
+from fee_lab_to_nwb.scherrer_ophys import ScherrerOphysNWBConverter
+from utils import get_timestamps_from_csv
 
-metadata_path = Path(__file__) / "scherrer_ophys_metadata.yaml"
+# The base folder path for ophys and behavior data
+ophys_dataset_path = Path("../../scherrer_ophys_data/")
+ophys_dataset_timestamp = "2021-06-03T11_46_29"
+
+ophys_dataset_session_id = ophys_dataset_path.stem + "_" + ophys_dataset_timestamp
+nwbfile_path = ophys_dataset_path / f"{ophys_dataset_session_id}.nwb"
+
+behavior_data_file_path = (
+    ophys_dataset_path / f"home_pos-speed-in_{ophys_dataset_timestamp}.csv"
+)
+behavior_movie_file_path = (
+    ophys_dataset_path / f"home_arena_{ophys_dataset_timestamp}.avi"
+)
```

Comment:
@CodyCBakerPhD Bit of confusion here related to `dandi organize`, I have all files in the same folder, do you want me to create a subfolder within `scherrer_ophys_data/` for the behavior video? Also for specifying a relative path, is it not correct? `"../../scherrer_ophys_data/"`

Reply:
> Bit of confusion here related to dandi organize

Sorry for the confusion, I was mentioning as a general practice not with respect to `dandi organize`


> Also for specifying a relative path, is it not correct? "../../scherrer_ophys_data/"

"../../scherrer_ophys_data/" is indeed a relative path, but I was saying it should be adjacent to the NWBFile for maximum safety and convenience. That is, the external file path should be "./home_arena_etc.avi", so that way, whenever you share the NWBFile, they just have to place it in the same folder as the movie file and it will then be correct.

Reply:
ah okay, thanks!